quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliCreateJvmApplicationIT.java

```diff
@@ -219,6 +220,18 @@ public void shouldCreateJacocoReportsFromApplicationOnJvm() {
                 "JaCoCo exec file doesn't exist");
     }
 
+    @Tag("QUARKUS-1296")
+    @Test
+    public void verifyRestEasyReactiveAndClassicResteasyCollisionUserMsg() {
+        QuarkusCliRestService app = cliClient.createApplication("dependencyCollision",
+                defaults().withExtensions("resteasy", "resteasy-reactive"));
+
+        CompletableFuture.runAsync(() -> app.start());
```

Comment:
```suggestion
        app.buildOnJvm();
```
Just `build` is enough.

Also, what is the point of running it asynchronously?

Reply:
Basically, this app is going to fail at start-time and we want to double-check that is failing because there are a collision between `resteasy` and `resteasy-reactive`

`app.start()` should be async because we need to run the app and check the logs even if the app fails at start-time.